src/server.tsx

```diff
@@ -72,20 +73,20 @@ interface BannerDataResponse {
     debug?: Debug;
 }
 
-const [, fetchConfiguredEpicTestsCached] = cacheAsync(
-    fetchConfiguredEpicTests,
-    60,
-    'fetchConfiguredEpicTests',
-);
-
 const buildEpicData = async (
     pageTracking: EpicPageTracking,
     targeting: EpicTargeting,
+    type: EpicType,
     params: Params,
     baseUrl: string,
 ): Promise<EpicDataResponse> => {
+    const [, fetchConfiguredEpicTestsCached] = cacheAsync(
```

Comment:
This will call `cacheAsync` for each request.
Let's instead add a cache for each EpicType to the file where `fetchConfiguredEpicTests` lives.
How about:
`const cachedEpicTests: {[key in EpicType]: Promise<EpicTests>} = {...}`

Reply:
🙈 good spot! I've moved it out now 👍